tests/python/relay/test_tidl.py

```diff
@@ -0,0 +1,210 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Unit tests for TIDL compilation."""
+import os
+import sys
+
+import argparse

```

Comment:
We shouldn't have arguments in the test script, you can make these argument in the tests methods with default values:

`def test_tidl_classification(batch_size=1, num_tidl_subgraphs=1, force_arm_only=False)`

We should consider adding an example/tutorial script.

Reply:
Arguments were removed. I will add a tutorial script.